content/blog/2023-04-10-is-latest-patch.html

```diff
@@ -0,0 +1,56 @@
+---
+title: "COVIDcast Fault Patch: JHU-CSSE is_latest Data"
+author: Nolan Gormley
+date: 2023-04-01
+tags:
+  - COVIDcast
+authors:
+  - nolan
+heroImage: blog-is-latest-numbers.jpeg
+heroImageThumb: blog-is-latest-numbers-thumb.jpeg
```

Comment:
...maybe Minh swapped the source values locally but didn't commit the results? because these look correct